src/modules/returns/lib/xml-adapter/mapper.js

```diff
@@ -76,7 +76,7 @@ const getMeterDetails = (ret) => {
 }
 
 const wasMeterUsed = (meterUsage) => {
-  return meterUsage.getAttribute('WasMeterUsed').value() === 'Y'
+  return meterUsage.attr('WasMeterUsed').value() === 'Y'
```

Comment:
The only functional code change needed when the package was swapped. Tests still pass...